config/initializers/reserved_words.rb

```diff
@@ -212,6 +212,15 @@ class ReservedWords
     welcome
     work
     yes
+    images
```

Comment:
**nitpick (non-blocking):** Should we keep this list in alphabetical order? Apart from being really good for my OCD, it would also make it easier to scan the list when adding new words later.